### PR TITLE
Create navbar

### PR DESCRIPTION
**Pull request by theninjawoohoo**
Added a navbar and logo to the site.

### CODE REVIEW

**Comment by theninjawoohoo** [comment]
[Create nav bar](https://trello.com/c/7NBGTCJs/15-create-nav-bar)

**Comment by theninjawoohoo** [comment]
[Creating A Logo](https://trello.com/c/RQOAn79W/18-creating-a-logo)

**Comment by theninjawoohoo** [comment]
image

File: client/src/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import './Navbar.css';
+import betterlogo from './image/betterlogo.png';
```
Comment (isose) [inline client/src/Navbar.js]: rename to logo

File: client/src/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+let devUrl = '/';
+
+function Navbar() {
+  return (
+    <div className="nav-bar">
+      <div className="div-center">
```
Comment (isose) [inline client/src/Navbar.js]: remove unused className

File: client/src/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        <a href={devUrl}>
+          <img alt="logo" src={betterlogo} className="logo"></img>
```
Comment (AndyLiuCodes) [inline client/src/Navbar.js]: We should use a better file name such as logo.png

File: client/src/Navbar.css [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  opacity: 90%;
+}
+
+.logo {
+  width: 260px;
+  height: 36px;
+  margin-left: 1em;
+}
+
+.home-text {
```
Comment (isose) [inline client/src/Navbar.css]: remove .home-text

File: client/src/Navbar.js (hunk 2 above)
Comment (AndyLiuCodes) [inline client/src/Navbar.js]: Remove div-center

File: client/src/Navbar.js (hunk 3 above)
Comment (AndyLiuCodes) [reply client/src/Navbar.js]: remove old logo.png and replace it with new logo

**Review by AndyLiuCodes — CHANGES_REQUESTED** [review]

**Review by isose — CHANGES_REQUESTED** [review]

**Review by AndyLiuCodes — APPROVED** [review]
LGTM

**Review by isose — APPROVED** [review]
Good!

**Pull request merged**